### ISSUE TITLE
Fatal Error

### ISSUE THREAD

**Issue by CashWilliams** [cthos/atom-gitbook#60]
1. clone repo into packages
2. apm rebuild
3. open atom

**Atom Version**: 1.5.3
**System**: Mac OS X 10.11.3
### Stack Trace

Failed to load the atom-gitbook package

```
At Cannot find module 'atom-space-pen-views'

Error: Cannot find module 'atom-space-pen-views'
    at Module._resolveFilename (module.js:336:15)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/cash/.atom/packages/atom-gitbook/lib/main.coffee:3:7)
    at Object.<anonymous> (/Users/cash/.atom/packages/atom-gitbook/lib/main.coffee:1:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:720:27)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:116:28
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:106:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:445:14)
    at PackageManager.module.exports.PackageManager.loadPackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:395:14)
    at AtomEnvironment.module.exports.AtomEnvironment.startEditorWindow (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:684:21)
    at module.exports (/Applications/Atom.app/Contents/Resources/app.asar/src/initialize-application-window.js:28:10)
    at setupWindow (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:86:5)
    at window.onload (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:41:9)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-gitbook, v0.1.0
jslint, v1.5.1

# Dev
No dev packages
```

**Comment by CashWilliams** [comment]
I had glanced over the _NPM Install_ section https://github.com/cthos/atom-gitbook#npm-install thinking it was a different way of installing, not continuing steps.

However, running `apm install` fatals as well:

```
$ apm install
Installing modules ✗
> buffertools@2.1.3 install /Users/cash/.atom/packages/atom-gitbook/node_modules/slug/node_modules/unicode/node_modules/bufferstream/node_modules/buffertools
> node-gyp rebuild

  CXX(target) Release/obj.target/buffertools/buffertools.o
  SOLINK_MODULE(target) Release/buffertools.node

> unicode@0.6.1 postinstall /Users/cash/.atom/packages/atom-gitbook/node_modules/slug/node_modules/unicode
> node install.js


dyld: lazy symbol binding failed: Symbol not found: _node_module_register
  Referenced from: /Users/cash/.atom/packages/atom-gitbook/node_modules/slug/node_modules/unicode/node_modules/bufferstream/node_modules/buffertools/build/Release/buffertools.node
  Expected in: dynamic lookup

dyld: Symbol not found: _node_module_register
  Referenced from: /Users/cash/.atom/packages/atom-gitbook/node_modules/slug/node_modules/unicode/node_modules/bufferstream/node_modules/buffertools/build/Release/buffertools.node
  Expected in: dynamic lookup
```

**Comment by CashWilliams** [comment]
Did some searching and tried: 

```
cd ~/.atom/packages/atom-gitbook
rm node_modules
xcode-select --install
apm rebuild
apm install
```

Still same error

**Comment by CashWilliams** [comment]
hum... so `npm install` worked.

**Comment by cthos** [comment]
Pretty sure it's related to what I blogged about here: https://www.alextheward.com/blog/apm-rebuild/

tl;dr I can only get it to rebuild on OSX using `npm rebuild` with the same version of node that Atom has bundled with it, namely `0.10.35`.
